src/operators.jl

```diff
@@ -708,6 +708,40 @@ egonet(g::AbstractGraph{T}, v::Integer, d::Integer, distmx::AbstractMatrix{U}=we
     g[neighborhood(g, v, d, distmx, dir=dir)]
 
 
+"""
+    induced_bipartite_subgraph(G,X,Y)
+
+Return the bipartite subgraph of `g` induced by the disjoint subsets of vertices X and Y.
+
+"""
+function induced_bipartite_subgraph(g::T,X::AbstractVector{U},Y::AbstractVector{U})  where T <: AbstractGraph where U <: Integer
+    
+    X ∩ Y != [] && throw("X and Y sould not intersect!")
```

Comment:
That `throw` here actually does throw a `String`, and not an `Exception` - this is a bit confusing, because Julia allows one to throw any kind of object. To demonstrate this:
```julia
julia> try
          throw("hello")
       catch e
          println(typeof(e))
       end
String
```
Instead you can either use `error("hello")` or use a subtype of `Exception` with `throw`. E.g. `throw(DomainError("hello"))`

Reply:
Thank you for the explanation!